tests/model_connectors/test_district_single_ghe.py

```diff
@@ -96,4 +96,7 @@ def test_simulate_district_system(self):
             f"{self.district._scaffold.project_name}.Districts.DistrictEnergySystem",
             run_path=self.district._scaffold.project_path,
             file_to_load=self.district._scaffold.package_path,
+            start_time=0,  # Day 0 (in seconds)
+            stop_time=3600,  # For 1 day duration (in seconds)
```

Comment:
```suggestion
            stop_time=86400,  # For 1 day duration (in seconds)
```

I think this is one hour... can we try 86400?

Reply:
My comment was incorrect, I really meant it to be one hour. Commit [34ca909](https://github.com/urbanopt/geojson-modelica-translator/pull/704/commits/34ca9093f8d97575ed27716e2cd9d2039e56badc) attempted one day of simulation and the CI runner failed. I have updated the comment.